src/Modules/Isrs.asm

```diff
@@ -370,12 +370,24 @@ t1_int_zero_rcp_checked_check_limit:
 
 t1_int_dynamic_pwm:
     ; Dynamic PWM
-    clr  C
     mov  A, Temp2
+
+    ; Choose between 96khz and 48khz
+    clr  C
+    subb A, Throttle_96to48_Threshold
+    jc   t1_int_run_96khz
+
+    ; Choose between 48khz and 24khz
+    clr  C
     subb A, Throttle_48to24_Threshold
     jc   t1_int_run_48khz
 
+
 IF PWM_CENTERED == 0
+
+; *******************
```

Comment:
I don't want to be a pain in the ass, but maybe we could keep the banners consistent?

```asm
;**** **** **** **** **** **** **** **** **** **** **** **** ****
; Edge aligned PWM
;**** **** **** **** **** **** **** **** **** **** **** **** ****
```

Reply:
Fixed